.github/workflows/screenshots.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     env:
       PERCY_PARALLEL_NONCE: ${{ github.run_id }}-${{ github.run_attempt }}
       PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+      PERCY_POSTINSTALL_BROWSER: false
```

Comment:
Prevents Percy downloading Chromium:

```console
[percy] Downloading Chromium 929511...
[percy] Successfully downloaded Chromium 929511
```

It can be skipped because we already download via Puppeteer:

https://github.com/alphagov/govuk-frontend/blob/760503320599cd644e19dc57c971cab492e6012f/shared/tasks/browser.mjs#L15